client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/Overlays/ViewingCone.cs

```diff
@@ -94,6 +94,14 @@ private set
       }
     }
 
+    private bool IsItInsideScaleWithoutLayerEnabled
```

Comment:
I think name should be a bit clearer and simpler.
Not sure what "It" means in this name. If it is camera, in that case name can be "IsCameraInsideScaleWithoutLayerEnabled".
But this part "WithoutLayerEnabled" might be also a bit unclear in the future.